### ISSUE TITLE
Feature request: Product Availibility easily turn on/off

### ISSUE THREAD

**Issue by mklimkiew** [PrestaShop/PrestaShop#19173]
BO > Catalog > Products

In the BO on the Products page you can turn on/off visibility of the product quite easily. 

If shop is not using quantity management then it would be useful to have option to quckly turn on/off **product availibility**. 
Now if I run out of stock I need to make a lot of clicks to turn off product availibility. And after the stock is again up I need to once again make a lot of clicks to turn on the product availibility.

How it could look like:

![status-1](https://user-images.githubusercontent.com/45269666/82139722-47456500-982a-11ea-84c3-e8c3da4e3df8.jpg)

**Comment by sowbiba** [comment]
Thanks @mklimkiew for the suggestion

If I understand well, you want the ON/OFF feature available when you edit a product but in the listing page ?

![Capture d’écran de 2020-05-18 10-59-43](https://user-images.githubusercontent.com/7426640/82207005-b25d6d00-98f8-11ea-971f-697f3b3795a4.png)

I will let @PrestaShop/prestashop-product-team study your request and come back to you asap

**Comment by sam-pires** [comment]
Thanks for the feedback @mklimkiew 👍 it will be taken into account for a rework on the product list page. 
What you want is to make the product unavailable but still visible, right ?

I have a couple of questions to better understand your needs:
- Why don't you set stocks for your product?
- How long is the period of time between the moment you have no stock and make the product unavailable and the moment you make it available again?

**Comment by mklimkiew** [comment]
> What you want is to make the product unavailable but still visible, right?

Yes that is right. This way it will be faster to manage products.


> * Why don't you set stocks for your product?

If run shop with low number of products it is easier not to use stocks. 

> * How long is the period of time between the moment you have no stock and make the product unavailable and the moment you make it available again?
In my case it is 5-10 days.

**Comment by pitkz** [comment]
"Why don't you set stocks for your product?"

Digital Products. I do not want to add fake stocks and check them regularly.
Custom Products, that are manufactured upon request. You dont need fake stocks either.
Companys without stock, like white labeling that is shipped Direct-2-Customer by Manufacture.
